jupyter/analysis-automation/BasicExample.ipynb

```diff
@@ -0,0 +1,287 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "jupyter": {
+     "source_hidden": true
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import requests\n",
+    "import asyncio\n",
+    "from systemlink.clients.nitdmreader.api.metadata_api import MetadataApi\n",
+    "from systemlink.clients.nitdmreader.models.file_list import FileList\n",
+    "\n",
+    "metadata_api = MetadataApi()\n",
+    "import scrapbook as sb\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "\n",
+    "from systemlink.clients.nitdmreader import *\n",
+    "data_api = DataApi()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "jupyter": {
+     "source_hidden": true
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# --- Define some usefull Meta Data Tool functions ---\n",
```

Comment:
Suggestion: Define useful metadata tool functions (unsure what a metadata tool is)

Reply:
Reworked comment to: "Metadata helper methods"